tests/meshcop/commissioner.cpp

```diff
@@ -398,6 +400,11 @@ bool Commissioner::IsValid(void) const
     return mCommissionState != kStateInvalid;
 }
 
+bool Commissioner::IsCommissionerAccepted()
```

Comment:
"All void functions or methods shall explicitly declare and specify the void type keyword."